community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/CostBasedPipeBuilderFactory.scala

```diff
@@ -38,7 +38,9 @@ object CostBasedPipeBuilderFactory {
             tokenResolver: SimpleTokenResolver = new SimpleTokenResolver(),
             maybeExecutionPlanBuilder: Option[PipeExecutionPlanBuilder] = None,
             plannerName: CostBasedPlannerName = PlannerName.default,
-            runtimeName: RuntimeName = RuntimeName.default
+            runtimeName: RuntimeName = RuntimeName.default,
+            useErrorsOverWarnings: Boolean = false,
+            showWarnings: Boolean = false
```

Comment:
I am +1 on this, but let's make this biggish changes in a separate PR, I don't like to defer the merge further.